pkg/pool-stable/contracts/StablePool.sol

```diff
@@ -14,7 +14,12 @@ import {
     IUnbalancedLiquidityInvariantRatioBounds
 } from "@balancer-labs/v3-interfaces/contracts/vault/IUnbalancedLiquidityInvariantRatioBounds.sol";
 import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
-import { SwapKind, PoolSwapParams, PoolConfig } from "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
+import {
```

Comment:
Could optionally do a straight import (as the last element of the interfaces section), once it goes multi-line and isn't an example. (Arguably, StablePool might be used as an example.)